integration-tests/docker/environment-configs/test-groups/shuffle-deep-store

```diff
@@ -19,5 +19,6 @@
 
 # Test with deep storage as intermediate location to store shuffle data
 # Local deep storage will be used here
+# If you are making a change in load list below, make the necessary changes in github actions too
```

Comment:
i wonder if it would be possible to feed these into the gha as an input of whether or not to run the test (not a blocker for this PR, im just a bit nervous about keeping these things synced)

Reply:
Yes. that would have been nice though will require a bit more work :) FWIW, the tests will run in full on a branch so it wouldn't be too long before we learn if something broke.